### ISSUE TITLE
[BUG] Translation not working

### ISSUE THREAD

**Issue by cfuent-es** [thedevdojo/voyager#4044]
## Version information

 - Laravel: v5.8.8
 - Voyager: v1.2.2
 - PHP: 7.2
 - Database: MySQL 5.7

## Description

The panel is not completely in English. Messages and buttons appear in another language. Print screen attached.
![voyager](https://user-images.githubusercontent.com/1443168/55081657-07984880-507f-11e9-966d-c89a5de71b03.png)

My voyager config lang:

```
'multilingual' => [
        /*
         * Set whether or not the multilingual is supported by the BREAD input.
         */
        'enabled' => false,

        /*
         * Select default language
         */
        'default' => 'en',

        /*
         * Select languages that are supported.
         */
        'locales' => [
            'en',
            //'pt-BR',
        ],
```

**Comment by MrCrayon** [comment]
Since multilingual is not enabled that part of config is not relevant, I think you should check:

1. Locale set on user profile
2. config/app.php fallback_locale and locale

**Comment by remluben** [comment]
I saw this issue by chance and as I installed a fresh new Voyager project a couple of minutes ago, I tested the language settings.

Laravel: v5.8.14
Voyager: v1.2.3
PHP: 7.3
Database: MySQL 5.7

As @MrCrayon  described, the multilingual option does not effect the language/locale settings at the users profile. In fact @fuentesbr mixed up multilingual settings (see https://docs.laravelvoyager.com/core-concepts/multilanguage) with user interface locale settings. 

I hope this helps.

Kind regards,
Benjamin
